awesomemotive/affiliatewp-external-referral-links
# Issue/46 - Bump WordPress minimum requirements

Issue #46


**Housekeeping**


**Proposed changes**


## How to test it
1. Install and activate External Referral Links add-on
2. Confirm the plugin works as expected without any PHP errors.
3. Confirm existence and test saving the new settings on `Settings > External Referral Links`
